pyproject.toml

```diff
@@ -48,6 +48,7 @@ lint.ignore = [
   "PLW0603",
   "PLW1510",
   "PLW2901",
+  "PLC0415",
```

Comment:
Is this supposed to be part of this change?

Reply:
You are likely using a different version of ruff.   You can use `pip install -f requireents-dev.txt` to install the one we use.  Or you can just revert this line an re-upload?